src/Simplex/Chat/Options.hs

```diff
@@ -55,9 +56,14 @@ chatOpts appDir =
           <> value defaultSMPServers
       )
     <*> switch
-      ( long "log"
+      ( long "connections"
+          <> short 'c'
+          <> help "Log every contact and group connection on start"
```

Comment:
same here
```suggestion
      ( long "subscriptions"
          <> short 'c'
          <> help "Log every contact and group subscription on start"
```

Reply:
subscription is our internal term, the current message in UI is "contact is connected"